### PR TITLE
Changelog 1.0.6.0

### PR DESCRIPTION
**Pull request by NicolasDorier**
@dennisreimann @Kukks @dstruck @pavlenex @rockstardev @bolatovumar @mariodian 

Please review, I would like to release a new version tomorrow night (japan time, in 24H)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+* Due to a privacy leak vulnerability, users of the payment button are strongly encouraged to update as soon as possible.
+
+### New features
+
+* Add QR code scan/show for PSBT  + Import wallet via QR [spec](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-005-ur.md) (supported by Cobo vault / Blue wallet) (#1931)
+* Checkout experience: Unified QR Code for on-chain and offchain payment (ref #2060) (@rockstardev)
+* Greenfield: Clean webhook API github-like (ref #2058) (@NicolasDorier @Kukks)
+* Greenfield: Can query invoice payment data (@Kukks)
+* Allow users to select block explorers from a list or specify their own URL  (@Kukks)
+* Allow disabling notifications per user and disabling specific notifications per user (ref #1991) (@Kukks)
```
Comment (Kukks) [inline Changelog.md:16]:
```suggestion
* Allow disabling live notifications globally and disabling specific notifications per user (ref #1991) (@Kukks)
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+* UI: Improvement of the checkout overlay (see [this comment](https://github.com/btcpayserver/btcpayserver/pull/1930#issuecomment-701298441)) (@dennisreimann)
+* BTCPay Server vault operations can now be retried without having to refresh the page (@NicolasDorier)
+* UX: Warning and hint system for stores not completely set up (@dennisreimann @rockstardev)
+* Greenfield (Breaking change): Invoice state renamed `Confirmed/Complete` to `Settled`. (@NicolasDorier)
+* Greenfield (Breaking change): Invoice state renamed `Paid` to `Processing`. (@NicolasDorier)
+* Breaking change: Remove SQLite as the default database option (@Kukks)
+* UI: Make sure transaction labels display correctly when there are many (ref #2076) (@ubolator)
+* UI: Properly center payment button content (@ubolator)
+* UI: Improvement of the lightning node info view (ref #2066) (@dennisreimann)
+* Share the link of a pay button so one can embbed in a QR code (fix #635) (@Kukks)
```
Comment (Kukks) [inline Changelog.md:33]:
```suggestion
* Share the link of a pay button so one can embed in a QR code (fix #635) (@Kukks)
```

**Review by Kukks — CHANGES_REQUESTED** [review]

**Comment by dennisreimann** [comment]
We have some [breaking changes](https://github.com/btcpayserver/btcpayserver/pull/2080/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R27-R29) in there – shouldn't wee assign the `1.0.6.0` version tag?

**Review by rockstardev — APPROVED** [review]
LGTM

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,63 @@
...
+* Checkout experience: Unified QR Code for on-chain and offchain payment (ref #2060) (@rockstardev)
+* Greenfield: Clean webhook API github-like (ref #2058) (@NicolasDorier @Kukks)
+* Greenfield: Can query invoice payment data (@Kukks)
+* Allow users to select block explorers from a list or specify their own URL  (@Kukks)
+* Allow disabling live notifications globally and disabling specific notifications per user (ref #1991) (@Kukks)
+* Allow custom redirect_url for PoS (ref #1924) (@mariodian)
+
+### Improvements
+
+* UI: Make store setup more awesome (ref #2011) (@dennisreimann @dstrukt)
```
Comment (pavlenex) [inline Changelog.md:21]: * UI: Make store setup more **intuitive** (ref #2011) (@dennisreimann @dstrukt)

File: Changelog.md [hunk 4]
```diff
@@ -1,5 +1,63 @@
...
+* UI: Make payment requests more awesome (ref #2011) (@dennisreimann @dstrukt)
```
Comment (pavlenex) [inline Changelog.md:22]: * UI: Improve payment request design (ref #2011) (@dennisreimann @dstrukt)

File: Changelog.md [hunk 5]
```diff
@@ -1,5 +1,63 @@
...
+* UI: Make pull payments more awesome (ref #2011) (@dennisreimann @dstrukt)
```
Comment (pavlenex) [inline Changelog.md:23]: * UI: Improve pull payments design (ref #2011) (@dennisreimann @dstrukt)

File: Changelog.md (hunk 4 above)
Comment (NicolasDorier) [reply Changelog.md:22]: okay will change mister-stick-in-the-ass

File: Changelog.md [hunk 6]
```diff
@@ -1,5 +1,63 @@
...
+* UI: Improvement of the checkout overlay (see [this comment](https://github.com/btcpayserver/btcpayserver/pull/1930#issuecomment-701298441)) (@dennisreimann)
```
Comment (pavlenex) [inline Changelog.md:24]: * UI: Improvement of the **modal** checkout overlay (see [this comment](https://github.com/btcpayserver/btcpayserver/pull/1930#issuecomment-701298441)) (@dennisreimann)

**Comment by dennisreimann** [comment]
```diff
- awesome changelog
+ improved changelog
```

😂

**Comment by rockstardev** [comment]
@NicolasDorier I missed formalists took over, next changelog we're bringing `awesome` back.

**Pull request merged**